app/server/sonicpi/lib/sonicpi/lang/core.rb

```diff
@@ -738,7 +738,7 @@ def osc(path, *args)
 " # Send an OSC messages with arguments to another program on the same machine
 
 use_osc \"localhost\", 7000        # Specify port 7000 on this machine
-osc \"/foo/bar\" 1, 3.89, \"baz\"  # Send an OSC message with path \"/foo/bar\"
+osc \"/foo/bar\", 1, 3.89, \"baz\" # Send an OSC message with path \"/foo/bar\"
                                    # and three arguments:
                                    # 1) The whole number (integer) 1
                                    # 2) The fractional number (float) 3,89
```

Comment:
While you're there @rbnpi - I notice that both the float value here and the one in the similar line below are written with a comma as the decimal separator - I don't think this is what Sam was intending either... you could change that too maybe.